js/src/modals/AddAddress/addAddress.js

```diff
@@ -98,64 +98,74 @@ export default class AddAddress extends Component {
     const { address, addressError, description, name, nameError } = this.store;
 
     return (
-      <Form>
-        <InputAddress
-          allowCopy={ false }
-          autoFocus
-          disabled={ !!this.props.address }
-          error={ addressError }
-          hint={
-            <FormattedMessage
-              id='addAddress.input.address.hint'
-              defaultMessage='the network address for the entry'
-            />
-          }
-          label={
-            <FormattedMessage
-              id='addAddress.input.address.label'
-              defaultMessage='network address'
-            />
-          }
-          onChange={ this.onEditAddress }
-          ref='inputAddress'
-          value={ address }
-        />
-        <Input
-          error={ nameError }
-          hint={
-            <FormattedMessage
-              id='addAddress.input.name.hint'
-              defaultMessage='a descriptive name for the entry'
-            />
-          }
-          label={
-            <FormattedMessage
-              id='addAddress.input.name.label'
-              defaultMessage='address name'
-            />
-          }
-          onChange={ this.onEditName }
-          ref='inputName'
-          value={ name }
-        />
-        <Input
-          hint={
-            <FormattedMessage
-              id='addAddress.input.description.hint'
-              defaultMessage='an expanded description for the entry'
-            />
-          }
-          label={
-            <FormattedMessage
-              id='addAddress.input.description.label'
-              defaultMessage='(optional) address description'
-            />
-          }
-          onChange={ this.onEditDescription }
-          ref='inputDescription'
-          value={ description }
-        />
-      </Form>
+      <ModalBox
+        icon={ <AddressesIcon /> }
+        summary={
+          <FormattedMessage
+            id='addAddress.header'
+            defaultMessage='To add a new entry to your addressbook, you need the network address of the account and can supply an optional description. Once added it will reflect in your address book.'
+          />
+        }
+      >
+        <Form>
+          <InputAddress
+            allowCopy={ false }
+            autoFocus
+            disabled={ !!this.props.address }
+            error={ addressError }
+            hint={
+              <FormattedMessage
+                id='addAddress.input.address.hint'
+                defaultMessage='the network address for the entry'
+              />
+            }
+            label={
+              <FormattedMessage
+                id='addAddress.input.address.label'
+                defaultMessage='network address'
+              />
+            }
+            onChange={ this.onEditAddress }
+            ref='inputAddress'
```

Comment:
Nope thinking of another one. No idea, not used there or in the tests, so unsure.